tests/items.test.js

```diff
@@ -73,18 +48,30 @@ describe('POST /items', function() {
 	});
 });
 
+
 describe('GET /items', function() {
+	request(app)
```

Comment:
1) use beforeAll() hook https://facebook.github.io/jest/docs/en/setup-teardown.html to do test setup
2) avoid duplicating code -> create a function that does login and use it in setup hook